src/main/java/org/mitre/synthea/world/agents/behaviors/planeligibility/GenericPayerEligibilty.java

```diff
@@ -0,0 +1,16 @@
+package org.mitre.synthea.world.agents.behaviors.planeligibility;
+
+import org.mitre.synthea.world.agents.Person;
+
+/**
+ * A payer eligibilty algorithm that accepts all patients.
+ */
+public class GenericPayerEligibilty implements IPlanEligibility {
```

Comment:
This class is only instantiated once. Given that it is an implementation of a single method interface, it can just be replaced with a lambda where it gets instantiated.

Reply:
I was able to do this for GenericPayerEligibility, but not for any of the others you pointed out since they don't use the default constructor and have fields.